CONTRIBUTING.md

```diff
@@ -14,214 +8,143 @@ Strapi is an open-source project administered by [the Strapi team](https://strap
+## Code of Conduct
 
-The Babel team wrote an excellent short post about [the pros and cons of the monorepo design](https://github.com/babel/babel/blob/master/doc/design/monorepo.md).
+This project and everyone participating in it is governed by the [Strapi Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please read the [full text](CODE_OF_CONDUCT.md) so that you can understand what actions will and will not be tolerated.
 
-We will do our best to keep the master branch as clean as possible, with tests passing all the times. However, it can happen that the master branch moves faster than the release cycle. To ensure you have the latest stable version, please refer to the [release on npm](https://www.npmjs.com/package/strapi).
+## Bugs
 
-If you send a pull request, please do it against the `master` branch. We are developing upcoming versions separately to ensure non-breaking changes from master to the latest stable major version.
+We are using [GitHub Issues](https://github.com/strapi/strapi/issues) to manage our public bugs. We keep a close eye on this so before filling a new task try to make sure the problem doesn't already exist.
 
-***
+## Get in Touch
 
-## Setup Development Environment
-To facilitate the contribution, we drastically reduce the amount of commands necessary to install the entire development environment. First of all, you need to check if you're using the [required versions of Node.js and npm](https://strapi.io/documentation/3.x.x/getting-started/install-requirements.html)
+- [Slack](https://slack.strapi.io/)
+- [Spectrum](https://spectrum.chat/strapi?tab=posts)
 
-**Note: Fish shell users** - due to the way fish shell deals with symlinks, the following steps will not work.
+---
 
-Then, please follow the instructions below:
+## Before Submitting a Pull Request
 
-#### 1. ▪️ Fork the repository
+The core team will review your pull request and will either merge it, request changes to it, or close it.
 
-[Go to the repository](https://github.com/strapi/strapi) and fork it to your own GitHub account.
+**Before submitting your pull request** make sure the following requirements are done:
 
-#### 2. 💿 Clone from your repository
+- Fork the repository and create your branch from `master`.
+- Run `yarn setup` in the repository root.
+- If you’ve fixed a bug or added code that should be tested, add tests and link the corresponding issue in either your commit or your PR!
+- Ensure the test suite passes:
+  - `yarn test:unit`
+  - `yarn test:front`
+- Make sure your code lints (`yarn lint`).
 
-```bash
-git clone git@github.com:YOUR_USERNAME/strapi.git
-```
+## Contribution Prerequisites
 
-#### 3. ⏳ Installation
+- You have [Node](https://nodejs.org/en/) at v8.0.0+ and [Yarn](https://yarnpkg.com/en/) at v1.2.0+.
```

Comment:
Node v10 only ?

Reply:
I agree node v10 for now, node 11 is not an LTS it's a dev version and node 12 isn't LTS until october: 
![image](https://user-images.githubusercontent.com/8593673/58448479-f9859900-80bc-11e9-81a5-4dc7540a6715.png)